src/test/fuzz/orphanage.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <random.h>
```

Comment:
nondeterminism does not work nicely with whitebox and greybox fuzzing. It makes it impossible for fuzz engines to track coverage precisely. Ideally the full program execution is fully predetermined by the fuzz input exclusively.

Please remove this and use `fuzzed_data_provider` instead.